package.json

```diff
@@ -62,15 +62,15 @@
     "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-promise": "^6.0.1",
     "fast-jwt": "^3.3.2",
```

Comment:
Worth upgrading fast-jwt to 4.0.0 to match @fastify/jwt's dependency on that version.